test/JDBC/input/BABEL-1683-vu-verify.sql

```diff
@@ -0,0 +1,70 @@
+-- NVARCHAR(128)
+INSERT INTO babel_1683_vu_prepare_table_nvarchar1 VALUES (1, NULL)
```

Comment:
unless there is specific reason, DDL/DML (insert/alter) should go in the prepare and query in verify part.

Reply:
Those insert are getting used to verify the behavior, I don’t think adding them in prepare will be useful

Reply:
still we should do few insert/alter in prepare. We verify two things in MVU test, MVU itself works and we can use object after MVU. For first, all that matters is what we do in prepare, so it is better to have some data on source DB